### PR TITLE
Web Crypto now needs a secure context

### PR DESCRIPTION
**Pull request by a2sheppy**
Web Cryptography now requires secure context
    Added a "secure_context" entry to the tables for
    the property and interfaces that needed it.
    The versions have been researched and other than
    Samsung Internet I am happy with. Samsung, I found
    no information on so left it false for the time
    being.
    
    Sources:
    * Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1333140
    * Chromium: https://codereview.chromium.org/2838603002 and https://storage.googleapis.com/chromium-find-releases-static/7d5.html#7d57a8fa0d13db4fcb178bb58bb9389fa7670d74
    * WebKit: https://bugs.webkit.org/show_bug.cgi?id=166959

### CODE REVIEW

File: api/Crypto.json [hunk 1]
```diff
@@ -180,6 +181,54 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "secure_context": {
```
Comment (bershanskiy) [inline api/Crypto.json]: `"secure_context_required"`, may be?

File: api/CryptoKey.json [hunk 2]
```diff
@@ -143,6 +143,54 @@
           }
         }
       },
+      "secure_context": {
```
Comment (bershanskiy) [inline api/CryptoKey.json]: `"secure_context_required"`?

File: api/SubtleCrypto.json [hunk 3]
```diff
@@ -722,6 +722,54 @@
           }
         }
       },
+      "secure_context": {
```
Comment (bershanskiy) [inline api/SubtleCrypto.json]: `"secure_context_required"`, may be?

File: api/Crypto.json (hunk 1 above)
Comment (vinyldarkscratch) [reply api/Crypto.json]:
```suggestion
        "secure_context_required": {
```

File: api/CryptoKey.json (hunk 2 above)
Comment (vinyldarkscratch) [reply api/CryptoKey.json]:
```suggestion
      "secure_context_required": {
```

File: api/SubtleCrypto.json (hunk 3 above)
Comment (vinyldarkscratch) [reply api/SubtleCrypto.json]:
```suggestion
      "secure_context_required": {
```

**Comment by Elchi3** [comment]
> Also, is `"experimental": true` a good call here? I don't think securing this API further is considered unstable or experimental.

This question still needs to be answered. I think we should set experimental to false.

**Pull request merged**